### PR TITLE
Add ability for user to utilize external DBs for Multi-DB sample

### LINKED ISSUE #690: Unable to run multiple-datasources as-is on local Spring Cloud Data Flow installation
Hi,
I cloned the repo to my hard disk and opened the multiple-datasources example with IntelliJ Idea. 
Without altering the source code, I ran mvn package to produce a *.jar.

I then registered the jar as an app using the Spring shell with this command:
```
app register --type task --name multiple-datasources --uri file://root/scdf/multiple-datasources-2.3.0-SNAPSHOT.jar
```
I created the task definition as shown:
Screenshot 2020-07-13 at 21 44 25

And tried to run it. Unfortunately, SCDF does not record start or stop times for the task execution, and looking into the logs vía the shell, I found the following relevant messages:

```
2020-07-22 23:00:59.298 ERROR 140 --- [           main] o.s.c.t.listener.TaskLifecycleListener   : An event to end a task has been received for a task that has not yet started.
2020-07-22 23:00:59.298  WARN 140 --- [           main] s.c.a.AnnotationConfigApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Failed to start bean 'taskLifecycleListener'; nested exception is java.lang.IllegalArgumentException: Invalid TaskExecution, ID 52 not found
```

I've looked at similar issues, but am unable to understand what I'm doing wrong. Moreover I was hoping that the example would just run "out of the box". What am I missing?

Thanks!

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #690

### CODE REVIEW

File: spring-cloud-task-samples/multiple-datasources/README.adoc [hunk 1]
```diff
@@ -21,9 +23,25 @@ which one to be used for the Spring Cloud Task repository.
...
+For example:
+[source,shell,indent=2]
+----
+export spring_datasource_url=<your db url>
+export spring_datasource_username=<your db user name>
+export spring_datasource_password=<your db user password>
+export spring_datasource_driverClassName=org.mariadb.jdbc.Driver
+export second_datasource_url=jdbc:<your db url>
+export second_datasource_username=<your db user name>
+export second_datasource_password=<your db user password>
```
Comment (benas) [inline spring-cloud-task-samples/multiple-datasources/README.adoc:45]: I think `export second_datasource_driverClassName=<your second db driver class name>` is missing here. Do you agree?

File: spring-cloud-task-samples/multiple-datasources/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,2 +1,4 @@
 spring.application.name=Demo Multiple DataSources Task
 logging.level.org.springframework.cloud.task=DEBUG
+
+spring.profiles.default=embedded
```
Comment (benas) [inline spring-cloud-task-samples/multiple-datasources/src/main/resources/application.properties:4]: IntelliJ is showing a warning here: `Cannot resolve configuration property 'spring.profiles.default' `. Should this be `spring.profiles.active=embedded` as in https://github.com/spring-cloud/spring-cloud-task/pull/727/files#diff-18da3e6827339e29aef887fbb5946d77bb4b38966bd0f636c83b0b37937a3a2fR38 ?

File: spring-cloud-task-samples/multiple-datasources/README.adoc [hunk 3]
```diff
@@ -21,9 +23,25 @@ which one to be used for the Spring Cloud Task repository.
 $ mvn clean package
 ----
 
-== Run:
+== Execute sample using 2 embedded databases (default):
 
 [source,shell,indent=2]
 ----
 $ java -jar target/multiple-datasources-2.2.2.RELEASE.jar
```
Comment (benas) [inline spring-cloud-task-samples/multiple-datasources/README.adoc:30]: This is not included in the PR, but the version of the jar in the command should be the same whether we use the `embedded` or `external` profile.

File: spring-cloud-task-samples/multiple-datasources/src/main/java/io/spring/configuration/ExternalDataSourceConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018-2019 the original author or authors.
```
Comment (benas) [inline spring-cloud-task-samples/multiple-datasources/src/main/java/io/spring/configuration/ExternalDataSourceConfiguration.java:2]: The year range should be `2018-2020` (same for other files).

File: spring-cloud-task-samples/multiple-datasources/README.adoc (hunk 3 above)
Comment (cppwfs) [reply spring-cloud-task-samples/multiple-datasources/README.adoc:30]: Updated.   Thanks for catching

File: spring-cloud-task-samples/multiple-datasources/README.adoc (hunk 1 above)
Comment (cppwfs) [reply spring-cloud-task-samples/multiple-datasources/README.adoc:45]: Ooops.   Thanks for catching

File: spring-cloud-task-samples/multiple-datasources/src/main/resources/application.properties (hunk 2 above)
Comment (cppwfs) [reply spring-cloud-task-samples/multiple-datasources/src/main/resources/application.properties:4]: Good catch.  Thanks!

File: spring-cloud-task-samples/multiple-datasources/src/main/java/io/spring/configuration/ExternalDataSourceConfiguration.java (hunk 4 above)
Comment (cppwfs) [reply spring-cloud-task-samples/multiple-datasources/src/main/java/io/spring/configuration/ExternalDataSourceConfiguration.java:2]: 2020 already? :-D   Thanks for catching

**Pull request closed without merging**